models/core/transfers.yml

```
@@ -0,0 +1,36 @@
+
+version: 2
+
+models:
+  - name: transfers
+    description: "Harmony transfer events"
```

Would be good to add full stop to all the descriptions to keep it consistent with others
```suggestion
    description: "Harmony transfer events."
```

---

Resolved in this commit: https://github.com/MetricsDAO/harmony_dbt/pull/19/commits/030cae378684c4f3ab40863164cfc74a48b76eb1